### PR TITLE
[BEAM-9196] Update testcontainers to 1.15.1

### PR DESCRIPTION
**Pull request by iemejia**
R: @piotr-szuberski @aromanenko-dev

### CODE REVIEW

**Comment by aromanenko-dev** [comment]
Run Java KafkaIO Performance Test

**Comment by aromanenko-dev** [comment]
Run Java HadoopFormatIO Performance Test

File: buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy [hunk 1]
```diff
@@ -448,7 +448,7 @@ class BeamModulePlugin implements Plugin<Project> {
     def slf4j_version = "1.7.30"
     def spark_version = "2.4.7"
     def spotbugs_version = "4.0.6"
-    def testcontainers_version = "1.15.0-rc2"
+    def testcontainers_version = "1.15.0"
```
Comment (bsideup) [inline buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy:451]: Consider using yesterday's 1.15.1 :)

```suggestion
    def testcontainers_version = "1.15.1"
```

**Comment by iemejia** [comment]
Thanks for the pointer @bsideup . Updated now

**Comment by piotr-szuberski** [comment]
LGTM

**Review by aromanenko-dev — APPROVED** [review]
Thanks, LGTM

**Pull request merged**